src/client/app/helpers/snippet.helper.ts

```diff
@@ -72,7 +73,6 @@ export const snippetFieldSortingOrder: { [key: string]: number } = {
 export function getSnippetDefaults(): ISnippet {
     return {
         id: '',
-        gist: '',
```

Comment:
Had forgotten to revert this change I'd made while I was testing out some code. Thanks for pointing it out. Fixed.